pulsar-client-kafka-compat/pulsar-client-kafka-tests/pom.xml

```diff
@@ -36,11 +36,6 @@
 
   <description>Tests to verify the correct shading configuration for the pulsar-client-kafka wrapper</description>
 
-  <properties>
```

Comment:
Why not remove this?

Reply:
I have verified the CI Unit action in my own forked repo, the config is not required.
My own PR: https://github.com/vissxu/pulsar-adapters/pull/4